CHANGELOG.txt

```diff
@@ -435,6 +438,8 @@
             - creme.forms.toImportField() is replaced by the widget 'ui-import-field' or 'creme.widget.ImportField'.
             - Full rework of creme.notification.NotificationBox.
             - creme.ajax.jqueryAjaxSend() arguments have changed from (url, data, successCb, errorCb, options) to ({url: url, body: data, ...options}, {done: successCb, fail: errorCb}).
+            - Removed creme.utils.confirmPOSTQuery(); Not really useful and almost never used.
+            - Removed creme.utils.confirmAjaxQuery(); Only used in confirmPOSTQuery.
```

Comment:
"Almost never used is scary", it seems you let some uses :sweat: 
Explaining how to migrate would be more useful.
I propose:
`In 'creme.utils', the functions 'confirmPOSTQuery()' & 'confirmAjaxQuery()' have been removed; use 'creme.utils.ajaxQuery' with the option "confirm: true" instead.`